src/formio.component.ts

```diff
@@ -161,6 +167,14 @@ export class FormioComponent implements OnInit {
             }
         });
     }
+    onPrevPage(data: any) {
+        this.alerts.setAlerts([]);
```

Comment:
Whyou couldn't we just add this one line to formio.wizard setPage method? I don't think this class should know anything about "pages"